felt-tool-api/src/main/java/io/github/feltmc/feltapi/mixin/tool/common/EnchantmentMixin.java

```diff
@@ -0,0 +1,19 @@
+package io.github.feltmc.feltapi.mixin.tool.common;
+
+import io.github.feltmc.feltapi.api.tool.extensions.EnchantmentExtension;
+import net.minecraft.enchantment.Enchantment;
+import net.minecraft.item.ItemStack;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfoReturnable;
+
+@Mixin(Enchantment.class)
+public class EnchantmentMixin {
+    @Inject(method = "isAcceptableItem", at = @At("HEAD"), cancellable = true)
+    public void felt_isAcceptableItem(ItemStack stack, CallbackInfoReturnable<Boolean> callback){
```

Comment:
generally don't like prefixing things with names at least not like this, wouldn't "acceptableItemMixin" me more appropriate?

Reply:
prefixing with the modid is just one of the ways to avoid any possible conflicts, it also identifies what mod injected what code

Reply:
i mean it's incredibly hard to have conflicts here is it not? considering its a method name not a class name or something, and does this actually get logged in any meanful way for the mod id to be here? even if so id rather not have the underscore in the name, better to have say like "feltIsAcceptableItemMixin"